src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,24 @@
 package mate.academy.bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
+    @NotEmpty
```

Comment:
https://www.baeldung.com/java-bean-validation-not-null-empty-blank
Read about diff between notEmpty and notBlank and choose what is more suitable for your case